src/app/_components/players/players-list/players-list.component.ts

```diff
@@ -31,6 +32,7 @@ export class PlayersListComponent {
   }
 
   public addPlayer() {
+    if(this.playerHelper.players.length === 23 ) return;
```

Comment:
23 magic number user enum peux être todo ?

Reply:
Need to change
this.playerHelper.players.length
By this.playerHelper.getPlayerNumber()
Don't understand the magic number